### ISSUE TITLE
Is it possible to implement multi-task learning based on mmdetection?

### ISSUE THREAD

**Issue by smilecode-github** [open-mmlab/mmdetection#5575]
I am trying to implement multi-task learning using mmdetection.

datasetA, datasetB (each different data type)

What I'm trying to implement is a shared backbone, datasetA uses only Head A, datasetB uses only Head B, and so on.

Backbone (shared) -> Head A (datasetA), Head B (datasetB) -> Sum (Loss A + Loss B) -> Loss

A simple flowchart is above.

Can this be implemented with current mmdetection and mmcv? Or is it more efficient to implement it from scratch?

In simple terms, I think it would be possible if you change the workflow well, but I'm not sure.

thank you.

**Comment by jshilong** [comment]
Sorry, there is no related implementation about multi-task learning, but You can implement it from scratch which I believe also is 
convenient

**Comment by jshilong** [comment]
Feel free to reopen the issue if there is any question

**Comment by hanchangdong** [comment]
> I am trying to implement multi-task learning using mmdetection.
> 
> datasetA, datasetB (each different data type)
> 
> What I'm trying to implement is a shared backbone, datasetA uses only Head A, datasetB uses only Head B, and so on.
> 
> Backbone (shared) -> Head A (datasetA), Head B (datasetB) -> Sum (Loss A + Loss B) -> Loss
> 
> A simple flowchart is above.
> 
> Can this be implemented with current mmdetection and mmcv? Or is it more efficient to implement it from scratch?
> 
> In simple terms, I think it would be possible if you change the workflow well, but I'm not sure.
> 
> thank you.

hello, have you solve is finally, i meet the same question, thks.

**Comment by alaa-shubbak** [comment]
I am also meet same issue ,any suggestions ?
